BUILD.bazel

```diff
@@ -1,4 +1,50 @@
 # TODO(#1532): Rename file to 'BUILD' post-Gradle.
+
+# Corresponds to being accessible to all Oppia targets. This should be used for production APIs &
+# modules that may be used both in production targets and in tests.
+package_group(
+    name = "oppia_api_visibility",
+    packages = [
+        "//...",
```

Comment:
Isn't this similar to `oppia_binary_visibility` below?

if not what packages does each cover, when do we need to use `oppia_api_visibility` & `oppia_binary_visibility`

Reply:
Hmm I tried making this clear in the documentation, but maybe it could use a bit more work. We'll generally be using API visibility unless something should only be accessible to the binary. I split them out to make the visibility groups themselves more explicit & to provide more control.